integration-tests/integration_test.go

```diff
@@ -37,7 +37,7 @@ const (
 
 func TestCollectorGRPC(t *testing.T) {
 	suite.Run(t, new(ProcessNetworkTestSuite))
-	suite.Run(t, new(BenchmarkCollectorTestSuite))
+	// suite.Run(t, new(BenchmarkCollectorTestSuite))
```

Comment:
[nit] Maybe remove this if we don't need it anymore.

Reply:
I'm unsure if we need this test at all anymore since it'll just run the ProcessNetworkTestSuite, which is duplicated elsewhere. I'll remove the whole thing.